Mathlib/MeasureTheory/Function/StronglyMeasurable/Basic.lean

```diff
@@ -462,6 +462,45 @@ protected theorem smul_const {𝕜} [TopologicalSpace 𝕜] [SMul 𝕜 β] [Cont
 #align measure_theory.strongly_measurable.smul_const MeasureTheory.StronglyMeasurable.smul_const
 #align measure_theory.strongly_measurable.vadd_const MeasureTheory.StronglyMeasurable.vadd_const
 
+/-- In a normed vector space, the addition of a measurable function and a strongly measurable
+function is measurable. Note that this is not true without further second-countability assumptions
+for the addition of two measurable functions. -/
+theorem _root_.Measurable.add_stronglyMeasurable
+    {α E : Type*} [MeasurableSpace α] [AddGroup E] [TopologicalSpace E]
```

Comment:
```suggestion
    {α E : Type*} {_ : MeasurableSpace α} [AddGroup E] [TopologicalSpace E]
```
(but the measurable space on E can stay an instance since it is a Borel space)